src/models/feedDao.ts

```diff
@@ -0,0 +1,242 @@
+import myDataSource from './index';
+
+const findUserAuth = async (userId: number) => {
+  let [userAuth] = await myDataSource.query(`
+    SELECT
+        sort_id, is_admin
+    FROM
+        users
+    WHERE
+        id = '${userId}'
+  `);
+
+  const userSortId = userAuth.sort_id;
+  const userAdminId = userAuth.is_admin;
+  userAuth = { userSortId, userAdminId };
+  return userAuth;
+};
+
+const isExistFeed = async (userId: number) => {
+  let [feed] = await myDataSource.query(`
+  SELECT
+      title
+  FROM
+      feeds
+  WHERE
+      user_id = '${userId}'
+  `);
+
+  const feedTitle = feed.title;
+  feed = feedTitle;
+  return feed;
+};
+
+const findBranchId = async (branch: string) => {
+  let [branchId] = await myDataSource.query(`
+    SELECT
+        id
+    FROM
+        branch
+    WHERE
+        branch_name = '${branch}'
+  `);
+
+  branchId = branchId.id;
+  return branchId;
+};
+
+const findCategoryId = async (category: string) => {
+  let [categoryId] = await myDataSource.query(`
+    SELECT
+        id
+    FROM
+        category
+    WHERE
+        category = '${category}'
+  `);
+
+  categoryId = categoryId.id;
+  return categoryId;
+};
+
+const createFeed = async (
+  userId: number,
+  categoryId: number,
+  title: string,
+  image: string,
+  introduction: string,
+  hompage: string,
+  detail_introduction: string,
+  member_benefit: string | string[],
+  contact: string | string[],
+  branchId: number
+) => {
+  await myDataSource.query(`
+    INSERT INTO feeds
+        (user_id, category_id, title, logo_img, introduction, website_url, detail_introduction, member_benefit, contact, use_branch_id, status_id)
+    VALUES
+        ('${userId}', '${categoryId}', '${title}', '${image}', '${introduction}', '${hompage}', '${detail_introduction}', '${member_benefit}', '${contact}', '${branchId}', '1')
+  `);
+};
+
+const findFeedId = async (title: string) => {
+  let [feedId] = await myDataSource.query(`
+    SELECT
+        id
+    FROM
+        feeds
+    WHERE
+        title = '${title}'
+  `);
+
```

Comment:
```suggestion
        title = ?
  `,
  [title]
  );
```

where 구문의 변수는 query injection 처리해도 좋을것 같아요